graphql/resolvers.go

```diff
@@ -357,6 +359,49 @@ func (r *queryResolver) TaskFiles(ctx context.Context, taskID string) ([]*Groupe
 	return groupedFilesList, nil
 }
 
+func (r *mutationResolver) SetPriority(ctx context.Context, taskID string, priority int) (*restModel.APITask, error) {
```

Comment:
since setting priority of a patch will need a resolver too, can this be named more specifically? `SetTaskPriority`